services/playlists/.env

```diff
@@ -0,0 +1 @@
+EXPRESS_PORT=8080
```

Comment:
I suggest remove .env configuration from public github repo and add it to .gitginore file. For example of config we can use .env.example